Demos/ReactiveKafkaFsProducer/src/main/scala/ReactiveKafkaFsProducer.scala

```diff
@@ -25,19 +26,22 @@ object ReactiveKafkaFsProducer extends App {
   val MAX_ALLOWED_FILES     = 1000
   val config                = ConfigFactory.load()
   val dir                   = config.getOrElse("dataDir", "").toOption.fold("")(identity(_))
+
   val path                  = File(s"${dir}/Casalini_mrc").path.toString
 
   val bootstrapServers = config.getOrElse("bootstrapServers", "").toOption.fold("")(identity(_))
 
   val producerSettings = ProducerSettings(system, new StringSerializer, new ByteArraySerializer)
     .withBootstrapServers(bootstrapServers)
 
-  val source: Source[Path, NotUsed] = Directory.ls(FileSystems.getDefault.getPath(path))
+  val path                          = inDir.path.toString
```

Comment:
This second definition of `path` is causing syntax errors

Reply:
Thanks, got it now.